onnx/large_helper.py

```diff
@@ -0,0 +1,346 @@
+# Copyright (c) ONNX Project Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import enum
+import os
+from typing import Any, Dict, Iterable, Optional, Set, Tuple
+
+import numpy as np
+
+from onnx import (
+    GraphProto,
+    ModelProto,
+    StringStringEntryProto,
+    TensorProto,
+    checker,
+    load_model,
+)
+from onnx.external_data_helper import (
+    ExternalDataInfo,
+    _get_all_tensors,
+    _sanitize_path,
+    uses_external_data,
+)
+from onnx.helper import make_model
+
+
+def make_large_tensor_proto(
+    location: str, name: str, tensor_type: int, shape: Tuple[int, ...]
+) -> TensorProto:
+    """
+    Create an external tensor.
+
+    :param location: unique identifier (not necessary a path)
```

Comment:
Why do we need a location at this point (unless it is really a path)? Specifically, why not use the tensor/initializer name as the key in the dictionary?

Reply:
I'm not sure about using the tensor name. Two subgraphs could use the same initializer name. So I prefered to give it a unique key. I could still append a unique suffix to the tensor name. Any recommandation?